### PR TITLE
Login is the default keychain name

### PR DESCRIPTION
**Pull request by dgyesbreghs**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context
Because macOS default KeyChain is called 'login', we can define a default value to the unlock_keychain action.
I've tested this with our own Jenkins instance.

### Description
Provide a default path value 'login.keychain' to the unlock_keychain action and make it optional.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/unlock_keychain.rb [hunk 1]
```diff
@@ -72,7 +72,8 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :path,
                                        env_name: "FL_UNLOCK_KEYCHAIN_PATH",
                                        description: "Path to the keychain file",
-                                       optional: false),
+                                       default_value: "login.keychain",
+                                       optional: true),
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/unlock_keychain.rb]: Can we change this to `optional: false`? We can still have a `default_value` but we don't want this to ever be `nil` 😊

**Review by joshdholtz — CHANGES_REQUESTED** [review]
One small change! Can we also add some tests to `unlock_keychain_spec.rb`?

File: fastlane/lib/fastlane/actions/unlock_keychain.rb (hunk 1 above)
Comment (dgyesbreghs) [reply fastlane/lib/fastlane/actions/unlock_keychain.rb]: Fixed it 🙂

**Comment by dgyesbreghs** [comment]
@joshdholtz I've created a new test to make sure everything is working.

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Can you delete the changes to `fastlane/spec/fixtures/xcodeproj/automatic_code_signing.xcodeproj/project.pbxproj`? Its on my list to prevent this change from happening 😉 Its an issue with our tests

**Comment by joshdholtz** [comment]
@dgyesbreghs Also, one quick question about this... Won't most login keychains still require the user to enter a password for the keychain to unlock it?

**Review by joshdholtz — APPROVED** [review]
Looks good! 👍

**Pull request merged**